as-ap-currinfo/as_ap_currinfo/as_ap_currinfo.py

```diff
@@ -70,7 +71,10 @@ def read(self, reason):
 
     def write(self, reason, value):
         """Write IOC pvs acording to main application."""
```

Comment:
user this PR to fix a typo here ('acording'...)

Reply:
ok!

Reply:
done!